controllers/comments.controllers.js

```diff
@@ -25,3 +26,14 @@ exports.postComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeComment(comment_id)
+    .then(() => {
+      res.status(204).send({});
+    })
```

Comment:
Hey Alex! As we are not sending a body back we could just send back the status. However, as the send method is the one that actually sends back the response, just `res.status` will not suffice. Take a look at the docs for the method `sendStatus`